src/providers/remoteConfig.ts

```diff
@@ -107,27 +111,49 @@ export interface TemplateVersion {
   /** When the template was updated in format (ISO8601 timestamp). */
   updateTime: string;
 
-  /** Metadata about the account that performed the update. */
+  /**
+   * Metadata about the account that performed the update, of
+   * type [`RemoteConfigUser`](/docs/reference/remote-config/rest/v1/Version#remoteconfiguser).
+   */
   updateUser: RemoteConfigUser;
 
-  /** A description associated with the particular Remote Config template. */
+  /** A description associated with this Remote Config template version. */
   description: string;
 
-  /** The origin of the caller. */
+  /**
+   * The origin of the caller, either the Firebase console or the Remote Config
+   * REST API. See [`RemoteConfigUpdateOrigin`](/docs/reference/remote-config/rest/v1/Version#remoteconfigupdateorigin)
+   * for valid values.
+   */
   updateOrigin: string;
 
-  /** The type of update action that was performed. */
+  /**
+   * The type of update action that was performed, whether forced,
+   * incremental, or a rollback operation. See
+   * [`RemoteConfigUpdateType`](/docs/reference/remote-config/rest/v1/Version#remoteconfigupdatetype)
+   * for valid values.
+   */
   updateType: string;
 
   /**
-   * The version number of the Remote Config template that was rolled back to,
-   * if the update was a rollback.
+   * The version number of the Remote Config template that was this update rolled
```

Comment:
This sounds awkward - maybe something like "If the update was a rollback, the version number of the Remote Config template it was rolled back to."?